content/sensu-go/5.2/_index.md

```diff
@@ -47,8 +47,8 @@ Sensu agents automatically register and de-register themselves with the Sensu se
 
 Get meaningful alerts when and where you need them.
 Use [event filters][18] to reduce noise and [check hooks][19] to add context and speed up incident response.
-Sensu integrates with the tools and services your organization already uses like [InfluxDB][6], [Slack][7], and more.
-Check out the [200+ plugins shared by the Sensu community][8], or write your own [Sensu Plugins][9] in any language.
+Sensu integrates with the tools and services your organization already uses like [PagerDuty][28], [AWS][29], and more.
```

Comment:
```suggestion
Sensu integrates with the tools and services your organization already uses like [PagerDuty][28], [InfluxDB][29], and more.
```

Reply:
I know it's not on bonsai at this moment, but I'd like to keep the influxdb example. The documentation for that collection of AWS plugins just doesn't pass muster. I am working on getting a release cut and onto bonsai.